### PR TITLE
chore: put request id to caching logs and log file checksum

### PR DESCRIPTION
**Pull request by blackrabbit99**
* update log information
* add config filecheck sum to understand when cache is stale on node
* pass request id to logs

### CODE REVIEW

File: ilc/server/app.js [hunk 1]
```diff
@@ -115,6 +132,17 @@ module.exports = (registryService, pluginManager) => {
             return;
         }
 
+        // temporary 20ms overhead
+        const hex = buildHashSum(JSON.stringify(JSON.stringify(registryConfig)));
```
Comment (b1ff) [inline ilc/server/app.js:136]: is double `stringify` intentional?

File: ilc/common/wrapWithCache.js [hunk 2]
```diff
@@ -63,17 +64,16 @@ const wrapWithCache =
...
                             }),
                         );
-
                         logger.info(
                             {
                                 memoName: name,
                                 hash,
                                 now,
                                 cachedAt: parsedItem ? parsedItem.cachedAt : undefined,
+                                id: contextStore && contextStore.get('reqId'),
```
Comment (b1ff) [inline ilc/common/wrapWithCache.js:73]: do you have an example of result log?

File: ilc/server/app.js (hunk 1 above)
Comment (blackrabbit99) [reply ilc/server/app.js:136]: NO. This is leftover. Fixing

Comment (blackrabbit99) [reply ilc/server/app.js]: Fixed

File: ilc/common/wrapWithCache.js (hunk 2 above)
Comment (blackrabbit99) [reply ilc/common/wrapWithCache.js:73]: The case is `contextStore` is not available on client side.

Comment (b1ff) [reply ilc/common/wrapWithCache.js:73]: hmm, `id` format is strange. I thought that we can use it per our internal log specification, but with such format is impossible.
Also please note that having `id` in logs might not be transparent without context of this PR

**Review by b1ff — APPROVED** [review]

**Review by yehor-manzhula — APPROVED** [review]

**Review by yehor-manzhula — APPROVED** [review]

**Pull request merged**